starknet/compiler.go

```diff
@@ -33,6 +35,7 @@ func Compile(sierra *SierraDefinition) (*CompiledClass, error) {
 
 	casmJSONOrErrorCstr := C.compileSierraToCasm(sierraJSONCstr)
 	casmJSONOrError := C.GoString(casmJSONOrErrorCstr)
+	// can we replace it with C.free call?
```

Comment:
I would advise against doing that. That string is allocated on Rust side, so it is safer to have it freed on Rust as well.